src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,8 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+        UserService loginCheck = new UserService();
```

Comment:
I would recommend to make UserService a class member. It will improve flexibility and simplify testing. Also, please consider better name, like `userService`.